src/app/styles/templates/_app.scss

```diff
@@ -47,8 +47,9 @@
         margin-right: 30px;
         margin-bottom: 30px;
         position: relative;
-        padding: 15px 20px;
+        padding: 15px 20px 18px;
         border-radius: 0;
+        height: 42px;
```

Comment:
@mtedeschi - Having the height set here caused the layer switcher to be pushed down so all the options weren't visible on the screen. In my last PR, I removed this height, but it looks like the minimize button is now on the bottom of the control.